demo/app/examples/FrameExample.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { useEffect, useRef, useState } from 'react';
+
+import HCaptcha from '../../../src/index.js';
+
+export function FrameExample({ document }) {
+  const captchaRef = useRef();
+
+  const executeCaptcha = async () => {
```

Comment:
should we remove these unused functions ?

Reply:
+ useEffect, useState

Reply:
addressed here [3f12a29](https://github.com/hCaptcha/react-hcaptcha/pull/185/commits/3f12a29d517a2821b4ef5bad72bd075b544fe5c9)